blacklight-cornell/app/views/advanced_search/_form.html.erb

```diff
@@ -0,0 +1,50 @@
+<% all_form_values = params['action'] == 'edit' ? params_to_form_values(params) : default_form_values(params) %>
+<% all_hidden_form_values = params['action'] == 'edit' ? params_to_hidden_form_values(params) : [] %>
```

Comment:
Huh, why did I do it this way. I could have just passed in the relevant values as locals when rendering the partials in index vs edit. I'll change this tomorrow.